isomorphe/app.py

```diff
@@ -70,6 +70,10 @@ def login():
         gn_info = migrator.gn.info()
     except (requests.exceptions.RequestException, GeonetworkConnectionError) as e:
         flash(f"Problème d'authentification ({e})", "error")
+        # still record the login info for the next try
```

Comment:
You don't get it again on the next try?

Reply:
The error? If you don't change anything, yes. But currently you'll loose everything you typed in the form, when you probably have to fix a thing or two.

Reply:
Sorry I meant the login/pass. Didn't realize you had to cache it manually so browser remembers it.

Reply:
Natively you might (?), but we're forcing the session values in the form to persist across runs.